packs/mmonit/actions/list_events.py

```diff
@@ -0,0 +1,43 @@
+from lib.mmonit import MmonitBaseAction
+
+
+class MmonitListEvents(MmonitBaseAction):
+    def run(self, **kwargs):
+        self.login()
+        data = {}
+        # Way too explicit for my taste but I guess its easier to understand
```

Comment:
Instead of using kwargs, can you please explicitly declare all the arguments in the method signature?

kwargs breaks method instrospection and many other things.